frontend/src/scenes/funnels/People.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react'
+import { useValues } from 'kea'
+import { funnelLogic } from './funnelLogic'
+import { Link } from 'lib/components/Link'
+import { percentage, Loading } from 'lib/utils'
+import { EntityTypes } from '~/types'
+import './FunnelPeople.scss'
+import { Card } from 'antd'
+
+export function People(): JSX.Element | null {
+    const { stepsWithCount, peopleSorted, peopleLoading, areFiltersValid } = useValues(funnelLogic({}))
+
+    return !stepsWithCount || !areFiltersValid ? null : (
```

Comment:
Code readability nit, maybe this is more clear?

```typescript
if (!stepsWithCount || !areFiltersValid) {
    return null
}
```

Reply:
Why not